.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: groovy
+os: linux
+dist: bionic
+arch: amd64
+sudo: required
+services: docker
+
+before_install:
+    - echo "This is the first attempt at .travis.yml for variantcalling"
+    - echo "This is an attempt to build the Dockerfile environment"
+    - docker build . --file ./Dockerfile --tag variantcaller
+    #- docker run -d --publish 8000:8080 variantcaller
+    - wget -qO- https://get.nextflow.io | bash 
+    - sudo mv nextflow /usr/local/bin
+
+script:
+    - echo $TRAVIS_BUILD_DIR
+    - ls $TRAVIS_BUILD_DIR
+    - nextflow run ./main.nf --align all --reads "CIN*R{1,2}*" --ref "./ref.fa" --fastp_min_base_quality 15 --fastp_max_percent_low_qual_base 40 --verbose -with-docker variantcaller;
```

Comment:
Please update the `test` profile in `nextflow.config` 
https://github.com/daviesdrew/variantcalling/blob/d452f8cc1a6901c818d997e4c4bbdc98af42c1a2/conf/test.config#L1-L26

The command-line for testing should be `nextflow run main.nf -profile test,docker`. You may need to specify the max number of CPUs and memory for Travis CI if that's the service you're going to go with.

Reply:
All data used in pipeline comes from test.config specification. I amcurrently trying to get the correct resources for snippy to work with travis build using nextflow run main.nf -profile test